tests/lib/rules/grapheme-string-literal.ts

```diff
@@ -0,0 +1,209 @@
+        String.raw`/[\q{\u{1f1f1}\u{1f1fb}|\u{1f1f1}\u{1f1fe}|\u{1f1f2}\u{1f1e6}|\u{1f1f2}\u{1f1e8}|\u{1f1f2}\u{1f1e9}|\u{1f1f2}\u{1f1ea}|\u{1f1f2}\u{1f1eb}|\u{1f1f2}\u{1f1ec}|\u{1f1f2}\u{1f1ed}|\u{1f1f2}\u{1f1f0}|\u{1f1f2}\u{1f1f1}|\u{1f1f2}\u{1f1f2}|\u{1f1f2}\u{1f1f3}|\u{1f1f2}\u{1f1f4}|\u{1f1f2}\u{1f1f5}|\u{1f1f2}\u{1f1f6}|\u{1f1f2}\u{1f1f7}|\u{1f1f2}\u{1f1f8}|\u{1f1f2}\u{1f1f9}|\u{1f1f2}\u{1f1fa}}]/v`,
+        String.raw`/[\q{\u{1f1f2}\u{1f1fb}|\u{1f1f2}\u{1f1fc}|\u{1f1f2}\u{1f1fd}|\u{1f1f2}\u{1f1fe}|\u{1f1f2}\u{1f1ff}|\u{1f1f3}\u{1f1e6}|\u{1f1f3}\u{1f1e8}|\u{1f1f3}\u{1f1ea}|\u{1f1f3}\u{1f1eb}|\u{1f1f3}\u{1f1ec}|\u{1f1f3}\u{1f1ee}|\u{1f1f3}\u{1f1f1}|\u{1f1f3}\u{1f1f4}|\u{1f1f3}\u{1f1f5}|\u{1f1f3}\u{1f1f7}|\u{1f1f3}\u{1f1fa}|\u{1f1f3}\u{1f1ff}|\u{1f1f4}\u{1f1f2}|\u{1f1f5}\u{1f1e6}|\u{1f1f5}\u{1f1ea}}]/v`,
+        String.raw`/[\q{\u{1f1f5}\u{1f1eb}|\u{1f1f5}\u{1f1ec}|\u{1f1f5}\u{1f1ed}|\u{1f1f5}\u{1f1f0}|\u{1f1f5}\u{1f1f1}|\u{1f1f5}\u{1f1f2}|\u{1f1f5}\u{1f1f3}|\u{1f1f5}\u{1f1f7}|\u{1f1f5}\u{1f1f8}|\u{1f1f5}\u{1f1f9}|\u{1f1f5}\u{1f1fc}|\u{1f1f5}\u{1f1fe}|\u{1f1f6}\u{1f1e6}|\u{1f1f7}\u{1f1ea}|\u{1f1f7}\u{1f1f4}|\u{1f1f7}\u{1f1f8}|\u{1f1f7}\u{1f1fa}|\u{1f1f7}\u{1f1fc}|\u{1f1f8}\u{1f1e6}|\u{1f1f8}\u{1f1e7}}]/v`,
+        String.raw`/[\q{\u{1f1f8}\u{1f1e8}|\u{1f1f8}\u{1f1e9}|\u{1f1f8}\u{1f1ea}|\u{1f1f8}\u{1f1ec}|\u{1f1f8}\u{1f1ed}|\u{1f1f8}\u{1f1ee}|\u{1f1f8}\u{1f1ef}|\u{1f1f8}\u{1f1f0}|\u{1f1f8}\u{1f1f1}|\u{1f1f8}\u{1f1f2}|\u{1f1f8}\u{1f1f3}|\u{1f1f8}\u{1f1f4}|\u{1f1f8}\u{1f1f7}|\u{1f1f8}\u{1f1f8}|\u{1f1f8}\u{1f1f9}|\u{1f1f8}\u{1f1fb}|\u{1f1f8}\u{1f1fd}|\u{1f1f8}\u{1f1fe}|\u{1f1f8}\u{1f1ff}|\u{1f1f9}\u{1f1e6}}]/v`,
+        String.raw`/[\q{\u{1f1f9}\u{1f1e8}|\u{1f1f9}\u{1f1e9}|\u{1f1f9}\u{1f1eb}|\u{1f1f9}\u{1f1ec}|\u{1f1f9}\u{1f1ed}|\u{1f1f9}\u{1f1ef}|\u{1f1f9}\u{1f1f0}|\u{1f1f9}\u{1f1f1}|\u{1f1f9}\u{1f1f2}|\u{1f1f9}\u{1f1f3}|\u{1f1f9}\u{1f1f4}|\u{1f1f9}\u{1f1f7}|\u{1f1f9}\u{1f1f9}|\u{1f1f9}\u{1f1fb}|\u{1f1f9}\u{1f1fc}|\u{1f1f9}\u{1f1ff}|\u{1f1fa}\u{1f1e6}|\u{1f1fa}\u{1f1ec}|\u{1f1fa}\u{1f1f2}|\u{1f1fa}\u{1f1f3}}]/v`,
+        String.raw`/[\q{\u{1f1fa}\u{1f1f8}|\u{1f1fa}\u{1f1fe}|\u{1f1fa}\u{1f1ff}|\u{1f1fb}\u{1f1e6}|\u{1f1fb}\u{1f1e8}|\u{1f1fb}\u{1f1ea}|\u{1f1fb}\u{1f1ec}|\u{1f1fb}\u{1f1ee}|\u{1f1fb}\u{1f1f3}|\u{1f1fb}\u{1f1fa}|\u{1f1fc}\u{1f1eb}|\u{1f1fc}\u{1f1f8}|\u{1f1fd}\u{1f1f0}|\u{1f1fe}\u{1f1ea}|\u{1f1fe}\u{1f1f9}|\u{1f1ff}\u{1f1e6}|\u{1f1ff}\u{1f1f2}|\u{1f1ff}\u{1f1fc}}]/v`,
+
+        String.raw`/[\q{竈|門|禰󠄀|豆|子}\q{煉󠄁|獄|杏|寿|郎}]/v`,
+        String.raw`/[\q{a|b|c|}]/v`,
+    ],
+    invalid: [
+        {
+            code: String.raw`/[\q{abc}]/v`,
+            errors: [
+                {
+                    message: "Use single grapheme in string literal.",
```

Comment:
The error message needs to be improved. If I didn't read the docs for the rule, I would have no idea what the error message wants me to do.

Maybe we could say this: "Only single characters and graphemes are allowed inside character classes. Use regular alternatives (e.g. `(?:foo|[...])`) for strings instead."

Reply:
Thank you! I will use that message.